clap_complete/src/generator/utils.rs

```diff
@@ -49,6 +49,10 @@ pub fn subcommands(p: &Command) -> Vec<(String, String)> {
         );
 
         subcmds.push((sc.get_name().to_string(), sc_bin_name.to_string()));
+
+        for a in sc.get_all_aliases() {
```

Comment:
As we are generally only doing visible aliases, can you switch this to `get_visible_aliases`?

Reply:
sure! on it!

i wasn't sure here because my thinking was that if someone was using an alias, visible or not, they should be getting auto completion still. would using `get_visible_aliases` break this behaviour? is it intended? (this is a discussion that can be moved to the issue(s) if necessary